docs/src/references/agent/tedge-configuration-management.md

```diff
@@ -216,6 +216,8 @@ Upon receiving a configuration update command, the agent performs the following
    1. It performs a `GET` request to the `tedgeUrl` specified in the command to retrieve the content.
    2. The agent then uses the `type` information (`mosquitto`) to to look up the target path from the `tedge-configuration-plugin.toml` file
    and applies the new configuration content to the corresponding `path`(`/etc/mosquitto/mosquitto.conf`).
+   If `tedge` user/group does not have write permissions to the path and its parent directory,
```

Comment:
Since this is just a reference doc, which is rarely checked by the user, it might be good to provide a similar reference from the user-focussed "operation doc" at `operate/c8y/configuration-management.md` as well.

Reply:
I considered it, but decided to only reference it in the _References_ section because the document under `operate/` is very high-level and `tedge-write` is an implementation detail of how the files are written to storage that the user normally doesn't have to be aware of. Document under `operate/` also links to this reference document of how the operation works, so I think this is the correct place for it.

Main point is to be searchable so a user that encounters `tedge-write` can easily find all the required details.

Reply:
> Main point is to be searchable so a user that encounters tedge-write can easily find all the required details.

Yeah, that I agree.